dashboard/test/models/pd/application/application_base_test.rb

```diff
@@ -47,5 +47,28 @@ class ApplicationBaseTest < ActiveSupport::TestCase
       application.reload
       assert application.pending?
     end
+
+    test 'regional partner name' do
+      partner = build :regional_partner
+      application = build :pd_facilitator1819_application, regional_partner: partner
+
+      assert_equal partner.name, application.regional_partner_name
+    end
+
+    test 'school name' do
+      school_info = build :school_info
+      teacher = build :teacher, school_info: school_info
+      application = build :pd_facilitator1819_application, user: teacher
+
+      assert_equal school_info.effective_school_name.downcase, application.school_name.downcase
```

Comment:
Since `school_name` returns the name titleized, why not leave that as it is and titleize the expected value to match for a more specific assertion?